config/webpack/environment.js

```diff
@@ -52,4 +53,16 @@ environment.loaders.delete('nodeModules');
 
 environment.loaders.append('erb', erb);
 
+if (process.env.HONEYBADGER_API_KEY) {
```

Comment:
should this be moved to `production.js` or do we want both dev and test to potentially upload this as well?